docs/src/main/paradox/routing-dsl/source-streaming-support.md

```diff
@@ -137,12 +134,20 @@ In this example, we want to consume this data in a streaming fashion from the re
 back pressure to the underlying TCP connection should the server be unable to cope with the rate of incoming data. Back pressure
 is automatically applied thanks to @extref[Akka Streams](akka-docs:scala/stream/index.html).
 
+Scala
+:   @@snip [JsonStreamingExamplesSpec.scala]($test$/scala/docs/http/scaladsl/server/directives/JsonStreamingExamplesSpec.scala) { #measurement-format }
+
+Java
+:   @@snip [JsonStreamingExamplesTest.java]($test$/java/docs/http/javadsl/server/JsonStreamingExamplesTest.java) { #measurement-format }
```

Comment:
Moved the marshalling code for `Measurement` here:

![2018-07-30_00h31_43](https://user-images.githubusercontent.com/7414320/43367928-f62b4ae0-938f-11e8-80f3-d43f6d687cbd.png)
![2018-07-30_00h31_50](https://user-images.githubusercontent.com/7414320/43367929-f64fe4ae-938f-11e8-97d4-c97be59aa6bd.png)